service_catalog/models/approval_step_state.py

```diff
@@ -60,7 +60,11 @@ def get_scopes(self):
         return self.approval_workflow_state.get_scopes()
 
     def who_can_approve(self):
-        return self.approval_workflow_state.request.instance.quota_scope.who_has_perm(self.approval_step.permission.permission_str)
+        return self.approval_workflow_state.request.instance.quota_scope.who_has_perm(
+            self.approval_step.permission.permission_str)
+
+    def who_can_accept(self):
+        return self.who_can_approve()
```

Comment:
May be rename who_can_approve to who_can_accept instead

Reply:
I prefer to keep the approve for a step and accept for request